app/models/practice_editor.rb

```diff
@@ -16,7 +16,9 @@ class PracticeEditor < ApplicationRecord
   def self.create_and_invite(practice, user)
     # Email param ensures the user associated with the practice editor has a valid va.gov email address
     self.create!(practice: practice, user: user, email: user.email)
-    PracticeEditorMailer.invite_to_edit_practice_email(practice, user).deliver
+    unless Rails.env == "development"
+      PracticeEditorMailer.invite_to_edit_practice_email(practice, user).deliver
+    end
```

Comment:
Hmm what do you think about checking the constant for DEV and STG prod envs as as well?

Reply:
@camillevilla I was under the impression that STG and DEV are both within the "production" environment at the application level

Reply:
oh I see what you're saying, you dont want emails sent out on DEV or STG either